### PR TITLE
Updated to reflect bot api 5.6

### PR DESCRIPTION
**Pull request by HeeroML**
The previous pull request had issues.

### CODE REVIEW

**Review by Loskir — APPROVED** [review]

File: site/docs/README.md [hunk 1]
```diff
@@ -84,5 +84,5 @@ Works! :tada:
 
 ---
 
-grammY supports the Telegram Bot API 5.5 which was [released](https://core.telegram.org/bots/api#december-7-2021) on December 7, 2021.
-(Last highlight: Responses to Join Requests)
+grammY supports the Telegram Bot API 5.6 which was [released](https://core.telegram.org/bots/api#december-30-2021) on December 30, 2021.
+(Last highlight: Spoiler and protected content support)
```
Comment (roj1512) [inline site/docs/README.md:88]:
```suggestion
(Last highlight: Spoiler and Improved Protected Content Support)
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

**Review by KnorpelSenf — APPROVED** [review]

File: site/docs/README.md (hunk 1 above)
Comment (KnorpelSenf) [reply site/docs/README.md:88]: This should be committed. Otherwise, I approve.

**Comment by KnorpelSenf** [comment]
@HeeroML are you still working on this?

**Comment by HeeroML** [comment]
Isn't this fixed already?

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
@WingLim sorry for the delay, now it's ready :)

**Review by WingLim — APPROVED** [review]
Done~

**Pull request merged**